pyproject.toml

```
@@ -216,7 +215,6 @@ commands =
     python3.9-dbg -m pytest -v --no-cov -R : -k "test_install"
 deps =
     pytest
-    pytest-leaks
     pytest-remotedata
```

I noticed that in the last working CI runs, `pytest-remotedata` was at version 0.3.3. When I tried installing `pytest-leaks` and `pytest-remotedata==0.3.3`, pytest would run properly again!

Try this:
```suggestion
    pytest-leaks
    pytest-remotedata<0.4.0
```